js/damage.js

```diff
@@ -696,6 +691,38 @@ function getDamageResult(attacker, defender, move, field) {
 			description.attackerAbility = attacker.ability;
 		}
 	}
+	if (move.dropsStats) {
+		if (move.usedTimes > 1) {
+			description.moveTurns = 'over ' + move.usedTimes + ' turns';
+		}
+		var droppedStats = move.dropsStats;
+		var boostDrop = Math.max(-6, attacker.boosts[attackStat] - (droppedStats * (move.usedTimes-1)));
+		if (attacker.ability === "Contrary") {
+			boostDrop = Math.min(6, attacker.boosts[attackStat] + (droppedStats * (move.usedTimes-1)));
+		}
+		var hasWhiteHerb = attacker.item === "White Herb";
+		var usedWhiteHerb = false;
+		for (var times = 0; times < move.usedTimes; times++){
+			var oldAttack = attack;
+			var newAttack = getModifiedStat(attacker.rawStats[attackStat], attacker.boosts[attackStat]);
+			damage = damage.map(function(affectedAmount) {
```

Comment:
Missing parentheses

Reply:
Uuuuuuh where are they missing ? Neither Sublime Text nor GitHub gave me anything that pointed at a mistake here, even though I could be wrong..

Reply:
Yeah I was actually just commenting never mind lol, it looked wrong on mobile at first